src/pages/Home/Home.jsx

```diff
@@ -28,11 +32,11 @@ function HomePage({ searchTerm }) {
   }, [searchTerm]);
 
   const HomeContainer = styled.section`
-    background-color: #fafafa;
+    margin-top: -4.3rem;
   `;
 
   return (
-    <HomeContainer>
+    <HomeContainer style={{ backgroundColor: darkMode ? '#303030' : '#fafafa' }}>
```

Comment:
You could also pass `darkMode` as a prop and handle the logic inside the styled component, something like this:

```css
backgroundColor: ${ props => props.darkMode ? '#303030' : '#fafafa' }
```

just to give an example